packages/react-components/react-search/src/components/SearchBox/useSearchBox.tsx

```diff
@@ -16,12 +22,28 @@ import { DismissRegular, SearchRegular } from '@fluentui/react-icons';
 export const useSearchBox_unstable = (props: SearchBoxProps, ref: React.Ref<HTMLInputElement>): SearchBoxState => {
   const { size = 'medium', disabled = false, contentBefore, dismiss, contentAfter, ...inputProps } = props;
 
+  const rootRef = React.useRef<HTMLDivElement>(null);
+  const searchBoxRef = React.useRef<HTMLInputElement>(null);
+
   const [value, setValue] = useControllableState({
     state: props.value,
     defaultState: props.defaultValue,
     initialState: '',
   });
 
+  const [focused, setFocused] = React.useState(false);
+
+  const onFocus = useEventCallback(() => {
+    setFocused(true);
+  });
+  const onBlur: React.FocusEventHandler<HTMLSpanElement> = useEventCallback(ev => {
+    if (rootRef.current!.contains(ev.relatedTarget)) {
```

Comment:
It is safer to conditionally call this

```suggestion
    if (rootRef.current?.contains(ev.relatedTarget)) {
```